apps/chat/src/store/conversations/conversations.epics.ts

```diff
@@ -407,6 +407,7 @@ const createNewConversationsEpic: AppEpic = (action$, state$) =>
       }) => {
         return state$.pipe(
           startWith(state$.value),
+          filter((state) => ModelsSelectors.selectIsRecentModelsLoaded(state)),
```

Comment:
```suggestion
          filter(ModelsSelectors.selectIsRecentModelsLoaded),
```

I suppose `state` has any type in the next map if we will return a callback